pkg/manifests/manifests.go

```diff
@@ -1657,6 +1657,9 @@ func (f *Factory) setupQueryLogFile(p *monv1.Prometheus, queryLogFile string) er
 		return nil
 	}
 	dirPath := filepath.Dir(queryLogFile)
+	if strings.HasPrefix(dirPath, ".") {
```

Comment:
`query.log` is valid (the Prometheus operator will automatically mount an emptyDir volume at `/var/log/prometheus`) but `filepath.Dir()` would return `.` for it.

```suggestion
	if dirPath != "." && strings.HasPrefix(dirPath, ".") {
```

Also worth adding a test case.

Reply:
Ohhh I didn't knew that! Thank you~